server/protocols/webadmin-cli/IMPLEMENT_WEBADMINCLI.md

```diff
@@ -0,0 +1,48 @@
+# Architecture Decision Record for new James CLI
```

Comment:
You architecture decision record needs to be in `src/adr` folder https://github.com/linagora/james-project/tree/master/src/adr

You need to have a number (42?) allocated to it.

You need to follow the same formalism than other ADRs.

You also need the 'Context' (what exist before, why we have problems with it) part, the 'Decision' part (what we will do), the 'Consequence' part, and finally some links (Reference).